dora/core/client/fs/src/main/java/alluxio/client/file/cache/store/QuotaManagedPageStoreDir.java

```diff
@@ -88,9 +97,21 @@ public long deletePage(PageInfo pageInfo) {
 
   @Override
   public void deleteTempPage(PageInfo pageInfo) {
+    if (mTempFileToPageInfoListMap.containsKey(pageInfo.getPageId().getFileId())) {
```

Comment:
`pageInfo.getPageId().getFileId()` is used several times in this function.
Do you think we can define a local variable for it?

Reply:
Updated.